### DISCUSSION TITLE
AppKit Biamp

### DISCUSSION THREAD

**Discussion by cyberbri24** [CeneroLLC/Cenero-App-Kit-Docs#6] [General]
Dialer or Source select objects, need a little more info when using Appkit volume and mute controls and Biamp Tesira. no help file for using a Tesira Dialer mod.

**Comment by ewilliams0305** [comment]
Thanks @cyberbri24 i will be sure to create some driver specific guides once the overall configuration and framework guides are completed. I will create an issue for tracking this.
